docs/SetupDatabaseForAWS.md

```diff
@@ -1,30 +1,175 @@
+
+It is recommended since the metrics and logs help you to investigate some issues in the production environment when they happen.
+
+#### Use VPC endpoint (Recommended in the production environment)
+
+// Note that We have not yet tested this feature with Scalar products.  
+// TODO: We need to test this feature with Scalar products.
+
+* [Using Amazon VPC endpoints to access DynamoDB](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/vpc-endpoints-dynamodb.html)
+
+It is recommended since the private internal connections not via WAN can make a system more secure.
+
+#### Configure Read/Write Capacity (Optional based on your environment)
+
+You can configure the **Read/Write Capacity** of DynamoDB tables based on your requirements. Please refer to the official document for more details on Read/Write Capacity.
+
+* [Read/write capacity mode](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.ReadWriteCapacityMode.html)
+
+You can configure Read/Write Capacity using ScalarDB/DL Schema Loader when you create a table. Please refer to the following document for more details on how to configure Read/Write Capacity (RU) using ScalarDB/DL Schema Loader.
+
+* [ScalarDB Schema Loader](https://github.com/scalar-labs/scalardb/blob/master/docs/schema-loader.md)
+
+## Amazon RDS for MySQL, PostgreSQL, Oracle, and SQL Server
+
+### Authentication method
+
+When you use RDS, you must set `JDBC_URL`, `USERNAME`, and `PASSWORD` in the ScalarDB/ScalarDL properties file as follows.
+
+```properties
+scalar.db.contact_points=<JDBC_URL>
+scalar.db.username=<USERNAME>
+scalar.db.password=<PASSWORD>
+scalar.db.storage=jdbc
+```
+
+Please refer to the following document for more details on the properties for RDS (JDBC databases).
+
+* [Getting Started with ScalarDB on JDBC databases](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb-on-jdbc.md)
+
+### Required configuration/steps
+
+#### Create an RDS database instance
+
+You must create an RDS database instance. Please refer to the official document for more details.
+
+* [Configuring an Amazon RDS DB instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_RDS_Configuring.html)
+
+### Optional configurations/steps
+
+#### Enable automated backups (Recommended in the production environment)
+
+You can enable automated backups. Please refer to the official document for more details.
+
+* [Working with backups](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_WorkingWithAutomatedBackups.html)
+
+It is recommended since the automated backups feature enables a point-in-time recovery feature. It can recover data to a specific point in time. It can reduce the downtime (pause duration) of backup operations when you use multi databases under Scalar products. Please refer to the following document for more details on how to backup/restore the Scalar product data.
```

Comment:
```suggestion
It is recommended since the automated backups feature enables a point-in-time recovery feature. It can recover data to a specific point in time. It can reduce downtime (pause duration) for backup operations when you use multi databases under Scalar products. Please refer to the following document for more details on how to backup/restore the Scalar product data.
```